### PR TITLE
Add combined_bce_rmse loss function

### PR DESCRIPTION
**Pull request by zhanyinx**
## Proposed changes

A new loss function has been added. The new loss combines the binary crossentropy loss for probability and rmse loss for spot locatisation.

## Types of changes

 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [X] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation content update
 - [ ] Other (please describe):

## Checklist

Put an x in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help!
This is simply a reminder of what we are going to look for before merging your code so nothing breaks.

 - [x] I have read the [contributing guidelines](https://deepblink.readthedocs.io/en/latest/contributing.html)
 - [x] Lint and unit tests pass locally with my changes (`tox`)
 - [x] I have added tests that prove my fix is effective or that my feature works
 - [x] I have added necessary documentation (if appropriate)
 - [ ] Any dependent changes have been merged and published in downstream modules
 - [ ] My commit messages format follow the project [git commit message format](https://github.com/slashsbin/styleguide-git-commit-message#commit-message-format)
 - [ ] First time only - I have added myself / the copyright holder to the authors.rst file

💔 Thank you!

### CODE REVIEW

**Review by BBQuercus — CHANGES_REQUESTED** [review]
* Change all `rsme` occurrences to `rmse` (also `combined_f1_rmse`).
* Add comment to describe what `score[0]` is?
* Add small description like the one for `combined_f1_rmse` why we're using `+` and `/10`

**Review by BBQuercus — APPROVED** [review]
Generally don't forget to `:sort` your imports alphabetically. But can be done another time.
